libraries/botbuilder-middleware-text-analytics/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@botbuildercommunity/text-analytics-middleware",
-  "version": "0.2.6",
+  "name": "@botbuildercommunity/middleware-text-analytics",
+  "version": "0.9.0",
```

Comment:
I  had this at 1.0, but bumped it down since we created a 1.0 branch. I can bump it back up, and we can assume the v1 branch is for moving and merging other packages. Open to suggestions.

Reply:
Changed to 1.0.0.